config/locales/en.yml

```diff
@@ -391,570 +408,624 @@ en:
-          - id: "0006"
-            text: "Get safety advice for survivors (Women’s Aid)"
-            href: "https://www.womensaid.org.uk/covid-19-coronavirus-safety-advice-for-survivors/"
-          - id: "0007"
-            text: "Get help from Nexus NI"
-            href: "https://nexusni.org/domestic-and-sexual-abuse-24-hour-helpline/"
-            show_to_nations:
-              - Northern Ireland
-          - id: "0008"
-            text: "Get help from Live Fear Free"
-            href: "https://gov.wales/live-fear-free"
-            show_to_nations:
-              - Wales
-          - id: "0009"
-            text: "Get help from Women’s Aid Scotland"
-            href: "https://womensaid.scot/"
-            show_to_nations:
-              - Scotland
-          - id: "0010"
-            text: "Get help from Scotland’s forced marriage and domestic abuse helpline"
-            href: "https://sdafmh.org.uk/"
-            show_to_nations:
-              - Scotland
-          - id: "0011"
-            text: "Get help for children (Childline)"
-            href: "https://www.childline.org.uk/"
-          - id: "0012"
-            text: "Get advice for children (Children’s Commissioner for Wales)"
-            href: "https://www.childcomwales.org.uk/coronavirus/"
-            show_to_nations:
-              - Wales
-          - id: "0013"
-            text: "Contact NSPCC if you’re concerned about a child"
-            href: "https://www.nspcc.org.uk/keeping-children-safe/our-services/nspcc-helpline/"
-          - id: "0014"
-            text: "Find helplines if you’re an older person (Older People’s Commissioner for Wales)"
-            href: "https://www.olderpeoplewales.com/en/coronavirus.aspx"
-            show_to_nations:
-              - Wales
-          - id: "0015"
-            text: "Find victim support helplines (this page is in Welsh)"
-            href: "https://www.victimsupport.org.uk/help-and-support/get-help/support-near-you/wales"
-            show_to_nations:
-              - Wales
+        - id: '0001'
+          text: If you’re in immediate danger call 999 and ask for the Police.
+        - id: '0002'
+          text: If you’re in danger and unable to talk on the phone, call 999, then
+            press 55 when prompted.
+        - id: '0003'
+          text: Call the National Domestic Abuse Helpline on <a href="tel:0808 2000
+            247" class="govuk-link">0808 2000 247</a>
+        - id: '0004'
+          text: If you’re a child or young person call Childline on <a href="tel:0800
+            1111" class="govuk-link">0800 1111</a>
```

Comment:
Should this this text should be split over two lines?

Not a change in this pull request I realise - does the phone number link needs a country code prepended to it?

```suggestion
          text: If you’re a child or young person call Childline on <a href="tel:+448001111" class="govuk-link">0800 1111</a>
```

Reply:
Good catch, will resolve this here and open a PR agianst master when this is merged!

Reply:
Checked that, confirmed it rendered ok:
![image](https://user-images.githubusercontent.com/3694062/82885053-81a2b680-9f3c-11ea-968e-33c161a60af4.png)

Had a minor freak out that the childline number broke, but then realised it's an oddly short number!

Reply:
See here for why i think we shouldn't change these: https://github.com/alphagov/govuk-coronavirus-find-support/pull/222/commits/b0c5b0aafdd33f6b983cf66d96c9d34c0e9bfcf6#r430281216